catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/NsiliCommonTestHelper.java

```diff
@@ -51,7 +51,7 @@
 import ddf.security.service.SecurityManager;
 import ddf.security.service.SecurityServiceException;
 
-public class NsiliCommonTest {
+public class NsiliCommonTestHelper {
```

Comment:
❓ Does making this NsiliTestCommon work? Otherwise I'm fine with this.

Reply:
Good catch. I'll change it out.